src/python/aqueduct_executor/operators/connectors/data/extract.py

```diff
@@ -145,4 +147,22 @@ class S3Params(models.BaseParams):
     merge: Optional[bool]
 
 
-Params = Union[RelationalParams, S3Params]
+class MongoDbFindParams(models.BaseParams):
+    kargs: Optional[List[Any]] = None
```

Comment:
another instance where it should be "args"? This isn't a MongoDB specific thing is it?